crates/core/src/lib.rs

```diff
@@ -431,8 +431,15 @@ impl SessionLivedBackend {
             duration.num_milliseconds().to_string()
         }).unwrap_or("".to_string());
 
+        let spawn_owner: String = self.metadata.labels
```

Comment:
It's probably worth exposing this as a `.owner()` method on the `SessionLivedBackend` itself. It would return an `Option<String>`, which would also simplify all the `unwrap_or()` chains because you could use the `?` operator. (Plus we will probably want to use it in other places, so it makes that easy)

Reply:
(There would still be one `unwrap_or` in this call, because it would still be returning an option and we want to log a string.)

Reply:
Ohh good call!

Reply:
Updated!